src/patternfly/demos/Card/card-content-1.hbs

```diff
@@ -0,0 +1,26 @@
+{{#> card-header}}
+  {{#> l-flex flex--modifier="pf-m-align-right"}}
+    {{#> l-flex flex--modifier="pf-m-grow"}}
+      {{#> brand brand--attribute='src="/assets/images/pf-logo-small.svg" alt="Logo"'}} 
+      {{/brand}}
+    {{/l-flex}}
+    {{#> dropdown id=(concat card--id "-dropdown-kebab-right-aligned") dropdown--IsActionMenu="true" dropdown-toggle--modifier="pf-m-plain" dropdown--HasKebabIcon="true" aria-label="Actions"}}
+    {{/dropdown}}
+    {{#> check}} 
+      {{#> check-input check-input--attribute=(concat 'id="' card--id '-check" name="' card--id '-check" aria-labelledby="' card--id '-check-label"')}}{{/check-input}}
+    {{/check}}
+  {{/l-flex}}
+{{/card-header}}
+{{#> card-body}}
+  {{#> content content--modifier="pf-m-no-margin"}}
+    <h4 id="{{concat card--id "-check-label"}}">Patternfly</h4>
+  {{/content}}
+  {{#> content}}
+    <small>Provided by Red Hat</small>
```

Comment:
Is this text appropriate for a `<small>` element or are you just using it to get the lighter text color?

> In HTML5, this element is repurposed to represent side-comments and small print, including copyright and legal text, independent of its styled presentation.

Reply:
Ah ok. I think this is a great use of the element.